src/ConnectKafkaFakeTestClass.cpp

```diff
@@ -17,8 +17,10 @@ bool ConnectKafkaFakeTestClass::CheckIfTopicExists(std::string Topic) {
   return Topic.compare("Topic that exists") == 0;
 }
 
-std::string ConnectKafkaFakeTestClass::Consume(std::string Topic) {
-  return "Subscribed";
+std::pair<std::string, bool>
+ConnectKafkaFakeTestClass::ConsumeFromOffset(std::string Topic) {
+  return std::pair<std::string, bool>("HiddenSecretMessageFromLovingNeutron",
+                                      true);
 }
 
 TopicMetadataStruct
```

Comment:
On line 28 you should probably initialise the `TopicMetadataStruct`, maybe something like:
```
TopicMetadataStruct MetadataStruct{"topic_name", {0}};
```